From a29cfe7f8f31ce003fa3e307d54d86a2a8e8cf19 Mon Sep 17 00:00:00 2001
From: Nuno Araujo <nuno.araujo@russo79.com>
Date: Mon, 21 Mar 2011 04:10:52 +0000
Subject: Fix compilation under Mono 2.10 (bgo#644516)

Force use of the overloaded implicit operator of Hyena.Gui.DragDropList
to solve mono 2.10 compilation problems.

Signed-off-by: Alexander Kojevnikov <alexander@kojevnikov.com>
---
diff --git a/src/PdfMod/Gui/DocumentIconView.cs b/src/PdfMod/Gui/DocumentIconView.cs
index 5796c72..386d662 100644
--- a/src/PdfMod/Gui/DocumentIconView.cs
+++ b/src/PdfMod/Gui/DocumentIconView.cs
@@ -319,7 +319,7 @@ namespace PdfMod.Gui
                 if (to_index < 0)
                     return;
 
-                var pages = args.SelectionData.Data as Hyena.Gui.DragDropList<Page>;
+                Hyena.Gui.DragDropList<Page> pages = args.SelectionData;
                 to_index -= pages.Count (p => p.Index < to_index);
                 var action = new MoveAction (document, pages, to_index);
                 action.Do ();
--
cgit v0.9

